kernel/src/scan/log_replay.rs

```diff
@@ -269,26 +235,17 @@ pub fn scan_action_iter(
     predicate: Option<ExpressionRef>,
 ) -> impl Iterator<Item = DeltaResult<ScanData>> {
     let mut log_scanner = LogReplayScanner::new(engine, table_schema, predicate);
-    let expression_handler = engine.get_expression_handler();
+    let expression = engine.get_expression_handler().get_evaluator(
```

Comment:
nit: maybe rename to `add_transform_expr`?

Reply:
That would cause the use site to become four lines instead of one due line length?